<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="s1" duration-ms="7084" started-at="2020-11-25T09:07:17 CST" finished-at="2020-11-25T09:07:24 CST">
    <groups>
    </groups>
    <test name="test1" duration-ms="7084" started-at="2020-11-25T09:07:17 CST" finished-at="2020-11-25T09:07:24 CST">
      <class name="com.webtest.demo.Admin_Login">
        <test-method status="PASS" signature="doBeforeClass()[pri:0, instance:com.webtest.demo.Admin_Login@6d8a00e3]" name="doBeforeClass" is-config="true" duration-ms="6196" started-at="2020-11-25T09:07:17 CST" finished-at="2020-11-25T09:07:23 CST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doBeforeClass -->
        <test-method status="FAIL" signature="testLogin1()[pri:0, instance:com.webtest.demo.Admin_Login@6d8a00e3]" name="testLogin1" duration-ms="97" started-at="2020-11-25T09:07:23 CST" finished-at="2020-11-25T09:07:23 CST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertTrue(Assert.java:43)
at org.testng.Assert.assertTrue(Assert.java:53)
at com.webtest.demo.Admin_Login.testLogin1(Unknown Source)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.TestNG.privateMain(TestNG.java:1336)
at org.testng.TestNG.main(TestNG.java:1304)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLogin1 -->
        <test-method status="PASS" signature="doAfterMethod_2()[pri:0, instance:com.webtest.demo.Admin_Login@6d8a00e3]" name="doAfterMethod_2" is-config="true" duration-ms="766" started-at="2020-11-25T09:07:23 CST" finished-at="2020-11-25T09:07:24 CST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doAfterMethod_2 -->
        <test-method status="PASS" signature="doAfterMethod()[pri:0, instance:com.webtest.demo.Admin_Login@6d8a00e3]" name="doAfterMethod" is-config="true" duration-ms="1" started-at="2020-11-25T09:07:24 CST" finished-at="2020-11-25T09:07:24 CST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doAfterMethod -->
      </class> <!-- com.webtest.demo.Admin_Login -->
    </test> <!-- test1 -->
  </suite> <!-- s1 -->
</testng-results>
